mulog-cloudwatch/src/com/brunobonacci/mulog/publishers/cloudwatch.clj

```diff
@@ -0,0 +1,106 @@
+(ns com.brunobonacci.mulog.publishers.cloudwatch
+  (:require [com.brunobonacci.mulog.publisher :as p]
+            [com.brunobonacci.mulog.buffer :as rb]
+            [com.brunobonacci.mulog.utils :as ut]
+            [cognitect.aws.client.api :as aws]
+            [cheshire.core :as json]
+            [cheshire.generate :as gen]))
+
+
+(defn- has-failures?
+  [rs]
+  (not (= {} rs)))
```

Comment:
I think this function should check for a more precise definition or error like a specific keyword